packages/dds/register-collection/src/test/consensusRegisterCollection.spec.ts

```diff
@@ -142,7 +142,7 @@ describe("ConsensusRegisterCollection", () => {
             });
 
             it("load", async () => {
-                const tree: ITree = buildTree(expectedSerialization);
+                const tree: ITree = buildTree(expectedSerialization) as ITree;
```

Comment:
Curious, why this cast is required?

Reply:
Not needed - figured out it was a protocol-base mismatch. Fixed in the latest iteration.